module-1/lab-bag-of-words/your-code/bonus.ipynb

```diff
@@ -22,14 +22,91 @@
   },
```

Comment:
Well done in bonus of bag_of_words!! As I said in functional programming lab when you have two levels for a list comprehension and you feel your code isn't clear just apply a double for loop.